core/trino-main/src/main/java/io/trino/server/security/oauth2/OAuth2Config.java

```diff
@@ -222,6 +223,19 @@ public OAuth2Config setPrincipalField(String principalField)
         return this;
     }
 
+    public Optional<String> getGroupsField()
+    {
+        return groupsField;
+    }
+
+    @Config("http-server.authentication.oauth2.groups-field")
```

Comment:
Please add changes to docs too

Reply:
It's already done:
```rst
   * - ``http-server.authentication.oauth2.groups-field``
     - The field of the access token used for Trino groups. The corresponding claim value must be an array.
```
Would like to add something?

Reply:
No. Thanks.